src/debug/gdb/gdb.c

```diff
@@ -430,12 +429,10 @@ static void put_packet(unsigned char *buffer)
 	} while (get_debug_char() != '+');
 }
 
-
 static void gdb_log_exception(char *message)
 {
 	while (*message)
 		put_exception_char(*message++);
-
 }
 
 /* Convert the memory pointed to by mem into hex, placing result in buf.
```

Comment:
@IulianOlaru249 I think this is fine for now. Anyhow notice that beside adding spaces around the operators you cleanup some newlines. Ideally this should be a separate patch in the same Pull Request.